.gitignore

```diff
@@ -33,3 +33,6 @@ csr.pem
 
 .DS_STORE
 
+# Ensuring Images are not uploaded in ./image , ./video path 
+.image
```

Comment:
This is incorrect. The directory needs to be maintained but no files can be uploaded using git.

Reply:
> The directory needs to be maintained but no files can be uploaded using git.
So should I also keep the copy of assets currently being used in the repo in the maintained directory as well?